presto-main/src/main/java/com/facebook/presto/spiller/FileSingleStreamSpiller.java

```diff
@@ -57,7 +58,6 @@
     static final int BUFFER_SIZE = 4 * 1024;
 
     private final Path targetFileName;
-    private final Closer closer = Closer.create();
```

Comment:
Moving closer creation to `close()` will conflict with some other change of mine in this class.